src/data/missions.csv

```diff
@@ -948,32 +948,32 @@
 "ユメヲカケル！を\n見よう","Watch 'Yume wo kakeru!'"
 "特定の編成でユメヲカケル！を\n見よう","Watch 'Yume wo kakeru! (special version)'"
 "涙ひかって明日になれ！を\n見よう","Watch 'Namida hikatte ashita ni nare!'"
-"BLAZEを\n見よう",""
+"BLAZEを\n見よう","Watch 'BLAZE'"
 "累計フォロー数25人を達成しよう","Achieve a cumulative total of 25 followers"
 "累計フォロー数50人を達成しよう","Achieve a cumulative total of 50 followers"
 "累計フォロワー数100人を達成しよう","Achieve a cumulative total of 100 followers"
 "累計フォロワー数200人を達成しよう","Achieve a cumulative total of 200 followers"
 "代表ウマ娘を他のトレーナーに\n育成で100回使ってもらおう","Have other trainers use your horse girl in training 100 times"
-"育成でURAファイナルズを\n大差勝ちで勝利しよう",""
-"ファン数を100万以上にして育成を完了しよう",""
+"育成でURAファイナルズを\n大差勝ちで勝利しよう","Win URA Finals by a large margin"
+"ファン数を100万以上にして育成を完了しよう","Complete a training run with over 1 million fans"
 "育成で河原に30回でかけよう",""
```

Comment:
I have no idea what this is, so leaving it unchanged

Reply:
Each training level has a name, differs by the level too; perhaps it's referring to a specific training with a specific level?

Reply:
Ooh, that could be. Do we have a list of these?

Reply:
We do, but unsure if it's really this lol
https://github.com/FabulousCupcake/umamusume-db-translate/blob/60d50e7d02aed781884d7eca45ba3cddc3b47f31/src/data/miscellaneous.csv#L52-L56

Reply:
Tried checking text_data, and there's no hits for 河原 besides the mission itself. Maybe this needs something else that's not in the game yet, since these are future missions after all.